cfme/containers/provider/openshift.py

```diff
@@ -240,7 +254,8 @@ def from_config(cls, prov_config, prov_key, appliance=None):
             image_repo=image_repo,
             image_reg=image_reg,
             image_tag=image_tag,
-            cve_loc=cve_loc
+            cve_loc=cve_loc,
+            virtualization=prov_config.get('virtualization', 'Disabled')
```

Comment:
I think this should default to None like metrics_type and alerts_type above, that way its just ignored/not filled when its not in the YAML.

Reply:
Changed.